wagtail/test/testapp/wagtail_hooks.py

```diff
@@ -382,12 +383,20 @@ class VariousOnDeleteModelViewSet(SnippetViewSet):
     inspect_view_enabled = True
 
 
+class SnippetChooserModelViewSet(SnippetViewSet):
+    model = SnippetChooserModel
+
+    list_display = ["__str__", "full_featured__text"]
```

Comment:
Sorry I forgot to suggest this before, could we add another case for multi-level relations? e.g.

```suggestion
    list_display = ["__str__", "full_featured__text", "full_featured__latest_revision__created_at"]
```

In the test, you'll need to do `ffs.save_revision()` so the `latest_revision` is populated. The method returns the `Revision` object, so you can hold on to it to get the `created_at` for assertion. You can then use `wagtail.utils.timestamps.render_timestamp` to assert the rendered timestamp is in the response. The label should be "Latest revision created at".